README.md

```diff
@@ -11,8 +11,9 @@ artifactory.user=email
 artifactory.password=xyz
 ```
 
-The file must contain the `sdk.token` provided by goTenna.
+The file must contain the `sdk.token` and `sdk.appid` provided by goTenna.
 
 ```
 sdk.token=
+sdk.appid=
```

Comment:
In the `depencies.gradle` we need to bump up the version number or this field will not work.

Reply:
once my sdk pr goes through and gets published, will update that here

Reply:
@rene-goTenna updated